PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+# Model Name
+
+## Use-cases
+Model use cases
+
+## Description
+Description of model
+
+## Model
+
+ |Model        |Download  | Download (with sample test data)|ONNX version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|Model Name       |ONNX Model download link with size  |  tar file containing ONNX model and synthetic test data|ONNX version used for conversion|Accuracy values |
+<hr>
+
+## Inference
+Step by step instructions on how to use the pretrained model and link to an example notebook/code. This section should ideally contain:
+
+### Input to model
+Input to network (Example: 224x224 pixels in RGB)
+
+### Preprocessing steps
+Preprocessing required
+
+### Output of model
+Output of network
+
+### Postprocessing steps
+Post processing and meaning of output
+<hr>
+
+## Dataset (Train and validation)
+This section should discuss datasets and any preparation steps if required.
+<hr>
+
+## Validation accuracy
+Details of experiments leading to accuracy and comparison with the reference paper.
+<hr>
+
+## Training
+Training details (preprocessing, hyperparameters, resources and environment) along with link to a training notebook (optional)
+<hr>
+
+## Validation
+Validation script/notebook used to obtain accuracy reported above along with details of how to use it and reproduce accuracy.
+<hr>
+
+## References
+Link to references
+<hr>
+
+## Contributors
+Contributors' name
+<hr>
+
+## Acknowledgements
```

Comment:
I recommend combining Contributors and Acknowlegements

Reply:
I feel combining Acknowledgements with References makes more sense because tags under acknowledgements is used by the contributor to help in the contribution of the model, just like content under references. What do you think @lupesko ?

Reply:
Agree with @abhinavs95 
This is the template for contributing models, and there is a difference between listing yourself as the contributor of the model, to the list of references/acks you list out (e.g. paper, reference implementations, etc.)
Suggest we simplify per @abhinavs95 's suggestion

Reply:
That's fine.

Reply:
merged Acknowledgements with References